<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>297</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>254</width>
    <height>297</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>254</width>
    <height>297</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>配置</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="btn_auto_config">
     <property name="text">
      <string>读取配置文件</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btn_save_config">
     <property name="text">
      <string>写入配置文件</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>端口</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_1">
     <property name="text">
      <string>COM1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboBox11"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>COM2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="comboBox21"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>COM3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="comboBox31"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>COM4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="comboBox41"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>COM5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="comboBox51"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>COM6</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="comboBox61"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>COM7</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QComboBox" name="comboBox71"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>COM8</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="comboBox81"/>
   </item>
   <item row="10" column="1">
    <widget class="QPushButton" name="btn_ok">
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
